Tests/test_PDB_parse_pdb_header.py

```diff
@@ -0,0 +1,50 @@
+"""Unit tests for PARTS of the parse_pdb_header module of Bio.PDB"""
+
+import unittest
+import warnings
+
+try:
+    import numpy
+    from numpy import dot  # Missing on old PyPy's micronumpy
```

Comment:
Ah - that probably was probably copied from ``test_PDB.py`` where things like SVD are being used. The ``dot`` import is probably irrelevant given how much PyPy has improved.

I'll check those - thanks!